js/ui/status/automaticUpdates.js

```diff
@@ -171,6 +199,11 @@ var Indicator = new Lang.Class({
 
         // The status label
         this._item.label.text = _("Automatic Updates");
+
+        this._state = this._getState();
```

Comment:
This line looks like it could be kept here, though (instead of moving it into the arrow function requested above)